charts/gradle-cache/templates/secret.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.configSecret.create }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "gradle-cache.configSecretName" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
nit: do we need this?

Reply:
Good catch. 
It's not necessary for sure. Forgot to remove after initialize chart.